pkg/cmd/cli/cmd/importimage.go

```diff
@@ -68,7 +69,7 @@ func RunImportImage(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, arg
 
 	from := cmdutil.GetFlagString(cmd, "from")
 	confirm := cmdutil.GetFlagBool(cmd, "confirm")
-
+	insecure := cmdutil.GetFlagBool(cmd, "insecure")
```

Comment:
I'd like this to match the annotation better (maybe `--insecure-repository`?) to clarify it is the repository that is insecure, not the API server